### PR TITLE
Rancher gear and proffesion

### PR DESCRIPTION
**Pull request by hyper2snyper**
```SUMMARY: Content "Adds a new scenario and profession"```

#### Purpose of change
I wanted to add a cowboy like profession.
#### Describe the solution
Added the two things I said I'm adding

### CODE REVIEW

File: data/json/scenarios/scenarios.json [hunk 1]
```diff
@@ -495,5 +495,15 @@
...
     "start_name": "Apartment Rooftop",
     "allowed_locs": [ "apartments_con_tower_114", "apartments_con_tower_014", "apartments_con_tower_104", "apartments_con_tower_004" ]
-  }
+  },
+  {
+    "type": "scenario",
+    "ident": "rancher",
+    "name": "Rancher",
+    "points": 4,
+    "description": "Youve lived on your ranch for most of your life, through hrad times and easy timess. One heck of a hard time is ahead..",
```
Comment (LaVeyanFiend) [inline data/json/scenarios/scenarios.json]:
```suggestion
    "description": "You've lived on your ranch for most of your life, through hard times and easy times. One heck of a hard time is ahead..",
```

File: data/json/professions.json [hunk 2]
```diff
@@ -3718,4 +3718,17 @@
       "female": [ "chestwrap" ]
     }
   }
+  {
+    "type": "profession",
+    "ident": "rancher",
+    "name": "Rancher",
+    "description": "You've raised cows or horses most your life, now, we'll see what happenes next.",
```
Comment (LaVeyanFiend) [inline data/json/professions.json]:
```suggestion
    "description": "You've raised cows or horses most your life, now, we'll see what happens next.",
```

File: data/json/items/armor/pets_horse_armor.json [hunk 3]
```diff
@@ -0,0 +1,19 @@
+{
+    "id": "saddlebag",
+    "type": "ARMOR",
+    "name": "Saddle Bags",
```
Comment (Rail-Runner) [inline data/json/items/armor/pets_horse_armor.json]:
```suggestion
    "name": "saddle bags",
```

File: data/json/professions.json (hunk 2 above)
Comment (Night-Pryanik) [reply data/json/professions.json]: Most of your life.

Comment (LaVeyanFiend) [reply data/json/professions.json]: I thought the missing "of" was an attempt to sound in-character, so I didn't bother to fix that one. 😛

**Comment by hyper2snyper** [comment]
How do I fix the conflicts?

**Pull request merged**